src/applications/vaos/new-appointment/index.jsx

```diff
@@ -117,6 +118,10 @@ function NewAppointmentSection({ isCernerOnlyPatient }) {
           component={DateTimeSelectPage}
         />
         <Route path={`${match.url}/va-facility`} component={VAFacilityPage} />
+        <Route
+          path={`${match.url}/va-facility-2`}
+          component={VAFacilityPageV2}
+        />
```

Comment:
Add route for new page that Lance created

Reply:
I don't know if this is really necessary, but one thing I think we can do with the new version of React Router is conditionally exclude routes. So you could in here add the feature flag to mapStateToProps and only render this route if it's true.